pajbot/constants.py

```diff
@@ -1 +1,2 @@
 VERSION = "1.55"
+CHARACTER_LIMIT = 500
```

Comment:
I just realized which file this was in - I also shouldn't be doing PR reviews when I'm awake!


Could you move this to `pajbot/tmi.py` instead?

Name and everything can stay, just stick it to the top or bottom of that file.

Reply:
Oh yeah, definitely! Just a minute~